### DISCUSSION TITLE
Retrieve external URL

### DISCUSSION THREAD

**Discussion by s267348** [microsoft/vscode-discussions#83] [Extension Development QnA] [answered]
If a user tabs vscode out (alt+tab or similar ways) and searches stuff online, when he tabs back to vscode is possible to save the last URL used during his research?
Or generally, is it possible to communicate in some way with an external browser from a vscode-extension?

**Comment by alefragnani** [comment, marked as answer]
Hi @s267348 ,

If you need VS Code to communicate with an external app, you will have to create a plugin/add-on for that external app as well. 

Maybe you are considering VS Code as a _kind of_ a Chrome app (because of its use of Electron), but it is not part of the Chrome browser you have installed.  Two different apps, two isolated sandboxes.

Hope this helps

**Reply by s267348** [reply]
Yes, thank you for the clarification!
